### ISSUE TITLE
[Analysis] Rate books condition based on images

### ISSUE THREAD

**Issue by GrayFlash** [AOSP-NIT-Surat/Bookstore-ML#2]
This issue is for you to add various possible approaches in docs, with the proper link to citations.

- [ ] rate the condition of a book on a linear scale from 1 to 5.

Suggest more features around it.

**Comment by shivambmishra10** [comment]
sir i want to confirm it that in this issue we have to pic the image of front page of book and then it our ml model would predict the rating , that only the issue is or something else?

**Comment by GrayFlash** [comment]
Yup, and for this issue you basically explain the approach you might want to take, and code is not required for this one.

**Comment by shivambmishra10** [comment]
1.Sir according to my knowing here we can use a fundamental algorithm logistic regression because we want output in discreate form such as categorical form  and via logistic we get output in this form only as per sigmoid function work in fram of probability and then setting cutoff we will convert it into discreate one. 
2.Next for image manipulation basics of open cv would work but not very sure.
3.then we will train the model on the basis of the data we have available and return output.
This is what i think. So please review it.

**Comment by vivektangudu2024** [comment]
I Think we have to train our model on 5 different datasets as we want to rate from 1-5, we will prepare a data set which contains ratings like which books come under 1 ,2.. now our model will relate this to output the grading

**Comment by GrayFlash** [comment]
yup can do that, but is there any data set for the same??

**Comment by vivektangudu2024** [comment]
i think to find a data set is difficult so we will give user to fix rate but max rate is MRP .customer who want to buy can have a chat with him and decide their price .
